### ISSUE TITLE
[BUG] git-lfs related Smudge error downloading arc-added lfs  

### ISSUE THREAD

**Issue by Brilator** [nfdi4plants/arcCommander#105]
**Describe the bug**
Indirectly related to arcCommander! Same error occurs with git only, but the referenced arc was init and synced via arccommander. 

I'm having troubles cloning via arc get: 
`../20220201_macos_DB/arc get -r ssh://git@gitlab.nfdi4plants.de:/brilator/samplearc_metabolomics.git`

Error message: 
```
Start processing command with the arguments:
        BranchName:Field ""
        RepositoryAddress:Field "ssh://git@gitlab.nfdi4plants.de:/brilator/samplearc_metabolomics.git"
 
Start Arc Get 
Cloning into 'samplearc_metabolomics'...
remote: Enumerating objects: 494, done.
remote: Counting objects: 100% (465/465), done.
remote: Compressing objects: 100% (446/446), done.
remote: Total 494 (delta 35), reused 433 (delta 19), pack-reused 29
Receiving objects: 100% (494/494), 185.19 KiB | 1.49 MiB/s, done.
Resolving deltas: 100% (38/38), done.
Downloading assays/Talinum_GCMS_minimal/dataset/150112_03.D/AcqData/71_82_EC5_2GHz_Fiensplitless_wo_sugar.ei.M/7890Method.bin (76 KB)
Error downloading object: assays/Talinum_GCMS_minimal/dataset/150112_03.D/AcqData/71_82_EC5_2GHz_Fiensplitless_wo_sugar.ei.M/7890Method.bin (2cf3723): Smudge error: Error downloading assays/Talinum_GCMS_minimal/dataset/150112_03.D/AcqData/71_82_EC5_2GHz_Fiensplitless_wo_sugar.ei.M/7890Method.bin (2cf37230ab009bb3c82e35052c3542ccfd5a5cb982afa712c97d4a14dcbe8905): batch request: missing protocol: "<unknown>"

Errors logged to /Users/dominikbrilhaus/Library/CloudStorage/OneDrive-SharedLibraries-DataPLANT/Tools - arcCommander/arcCommander_testing/20220209_tests_DB/samplearc_metabolomics/.git/lfs/logs/20220209T105413.96276.log
Use `git lfs logs last` to view the log.
error: external filter 'git-lfs filter-process' failed
fatal: assays/Talinum_GCMS_minimal/dataset/150112_03.D/AcqData/71_82_EC5_2GHz_Fiensplitless_wo_sugar.ei.M/7890Method.bin: smudge filter lfs failed
warning: Clone succeeded, but checkout failed.
You can inspect what was checked out with 'git status'
and retry with 'git restore --source=HEAD :/'

ERROR: Git command could not be run. 
Done processing command. 
```

- Note 1
developer arcCommander version (Feb 01 2022) used here

- Note 2
I cloned via ssh due to current macos issue with https git cloning https://stackoverflow.com/questions/70940973/error06fff089digital-envelope-routinescrypto-internalbad-key-length/70945773

- Note 3
`git clone ssh://git@gitlab.nfdi4plants.de:/brilator/samplearc_metabolomics.git` throws the same (git lfs part of the) error 

- Note 4 
Cloning an empty ARC without lfs bits works fine 
`../20220201_macos_DB/arc get -r ssh://git@gitlab.nfdi4plants.de:27005/brilator/emptyarc.git`

**OS and framework information (please complete the following information):**
- macOS Monterey v12.2
- git version 2.34.1
- git-lfs/3.0.2 (GitHub; darwin amd64; go 1.17.2)

**Comment by Brilator** [comment]
TLDR: 
Can someone comment, wether `arc get -r ssh://git@gitlab.nfdi4plants.de:/brilator/samplearc_metabolomics.git` is running on your system.

**Comment by HLWeil** [comment]
`arc get -r ssh://git@gitlab.nfdi4plants.de:/brilator/samplearc_metabolomics.git`

> Start processing command with the arguments:
>         BranchName:Field ""
>         RepositoryAddress:Field "ssh://git@gitlab.nfdi4plants.de:/brilator/samplearc_metabolomics.git"
> 
> Start Arc Get
> ERROR: Cloning into 'samplearc_metabolomics'...
> ERROR: Host key verification failed.
> ERROR: fatal: Could not read from remote repository.
> ERROR:
> ERROR: Please make sure you have the correct access rights
> ERROR: and the repository exists.
> Done processing command.

I'm not sure about the syntax you used for the repo address. I tried: 
`arc get -r https://gitlab.nfdi4plants.de/brilator/samplearc_metabolomics` an that worked fine

**Comment by HLWeil** [comment]
But that doesn't seem to be the problem anyways judging from your error message. 

Btw my `git` and `git-lfs` versions are the same

**Comment by Martin-Kuhl** [comment]
I also get an error using `arc get -r ssh://git@gitlab.nfdi4plants.de:/brilator/samplearc_metabolomics.git`

> remote: Enumerating objects: 494, done.
remote: Counting objects: 100% (465/465), done.
remote: Compressing objects: 100% (446/446), done.
remote: Total 494 (delta 35), reused 433 (delta 19), pack-reused 29R
Receiving objects: 100% (494/494), 185.19 KiB | 3.14 MiB/s, done.
Resolving deltas: 100% (38/38), done.
Downloading assays/Talinum_GCMS_minimal/dataset/150112_03.D/AcqData/71_82_EC5_2GHz_Fiensplitless_wo_sugar.ei.M/7890Method.bin (76 KB)
Error downloading object: assays/Talinum_GCMS_minimal/dataset/150112_03.D/AcqData/71_82_EC5_2GHz_Fiensplitless_wo_sugar.ei.M/7890Method.bin (2cf3723): Smudge error: Error downloading assays/Talinum_GCMS_minimal/dataset/150112_03.D/AcqData/71_82_EC5_2GHz_Fiensplitless_wo_sugar.ei.M/7890Method.bin (2cf37230ab009bb3c82e35052c3542ccfd5a5cb982afa712c97d4a14dcbe8905): batch request: missing protocol: "<unknown>"
Errors logged to C:\Users\Martin\Desktop\test\samplearc_metabolomics\.git\lfs\logs\20220209T113728.1721495.log
Use `git lfs logs last` to view the log.
error: external filter 'git-lfs filter-process' failed
fatal: assays/Talinum_GCMS_minimal/dataset/150112_03.D/AcqData/71_82_EC5_2GHz_Fiensplitless_wo_sugar.ei.M/7890Method.bin: smudge filter lfs failed
warning: Clone succeeded, but checkout failed.
You can inspect what was checked out with 'git status'
and retry with 'git restore --source=HEAD :/'
ERROR: Git command could not be run.
Done processing command.

Seems to work for `arc get -r https://gitlab.nfdi4plants.de/brilator/samplearc_metabolomics`. At least it's still running...

**Comment by HLWeil** [comment]
Might be an authentication problem? If I remember correctly, authentication to the `GitLab instance` and the `git lfs storage` behind it are separately handled. 
Maybe using your approach somehow the `ssh key` is handed to `GitLab` but not to `git lfs`?

**Comment by Brilator** [comment]
see my note 4: 
This `arc get -r ssh://git@gitlab.nfdi4plants.de:27005/brilator/emptyarc.git` works.

**Comment by HLWeil** [comment]
Ah didn't catch that. But I guess that supports my theory

**Comment by Brilator** [comment]
And as I said (note 2), there's some stupid Mac issue with cloning via https. 
However, this is not the case for cloning from GitHub. 
`arc get -r https://github.com/nfdi4plants/ARC.git`works.

**Comment by Brilator** [comment]
> Might be an authentication problem? If I remember correctly, authentication to the `GitLab instance` and the `git lfs storage` behind it are separately handled.
> Maybe using your approach somehow the `ssh key` is handed to `GitLab` but not to `git lfs`?

Nope, that's not it. 
Cloning another arc, from gitlab, with lfs files, via ssh, works fine: 

`arc get -r ssh://git@gitlab.nfdi4plants.de:27005/martinkuhl/samplearc_rnaseq.git`


So there's something particularly iffy about the samplearc_metabolomics with ssh and git lfs.

**Comment by HLWeil** [comment]
Could you clear out your issues? 

Because of `Note 3`, we can assume that this is not an issue with the `ArcCommander`, but with the `git <-> GitLab` interaction. Therefore, I think this would actually rather fall within the scope of tasks of the guys maintaining this `GitLab` instance.

**Comment by Brilator** [comment]
Yes. 
I pushed the repository to the .org gitlab (https://git.nfdi4plants.org/brilator/samplearc_metabolomics.git) and was able to `arc get -r` from there.
